challenge-04/README.md

```diff
@@ -29,35 +52,65 @@ seguintes propriedades (os valores devem ser do tipo mostrado abaixo):
 - `assentos` - Number - cinco por padrão
 - `quantidadePessoas` - Number - zero por padrão
 */
-?
+var carro =
+{
+    marca: 'Volkswagen',
+    modelo: 'Golf',
+    placa: 'JAM0J59',
+    ano: 2025,
+    cor: 'Azul',
+    quantasPortas: 4,
+    assentos: 5,
+    quantidadePessoas: 0,
+};
 
 /*
 Crie um método chamado `mudarCor` que mude a cor do carro conforme a cor
 passado por parâmetro.
 */
-?
+var novaCor
+carro.mudaCor = function cor(novaCor)
```

Comment:
Acho que não há necessidade das funções serem nomeadas e se for adicionar nome penso que o nome da função deva ser mais semântico. Por exemplo nesse caso `changeColor`